recipe/conda_build_config.yaml

```diff
@@ -104,9 +104,8 @@ rust_compiler:
 macos_machine:                 # [osx]
   - x86_64-apple-darwin13.4.0  # [osx and x86_64]
   - arm64-apple-darwin20.0.0   # [osx and arm64]
-MACOSX_DEPLOYMENT_TARGET:      # [osx]
-  - 11.0                       # [osx and arm64]
-  - 10.9                       # [osx and x86_64]
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+  - "10.13"                  # [osx and x86_64]
```

Comment:
OK, so this is actually a pretty serious mis-merge. I'm going to revert this commit right now.

It's an unfortunate consequence of the situation in https://github.com/conda-forge/conda-forge.github.io/issues/1844 that we cannot merge something like this yet without _a lot_ of infrastructure work, but I'm working on this with a high degree of urgency and trying to do it right.

In the meantime, please do not merge any changes to `MACOSX_DEPLOYMENT_TARGET` in the pinning repo (which unfortunately we require to be present for abseil/grpc/protobuf-related migrations).

CC @conda-forge/core

Reply:
Partial revert in 04441520c8b5b0bf23d9bca7d53dfa38eda0088d